WPFClient/Pages/Login.xaml.cs

```diff
@@ -28,33 +16,47 @@ public Login()
 
         private void loginButton_Click(object sender, RoutedEventArgs e)
         {
-            App._connection.InvokeCoreAsync("ReceiveUsername",
-                                            args: new[] { userNameTextBox.Text });
-            //App._connection.On("ReceiveChatMessage", (string message) =>
-            //{
-            //    WriteLine($"{ message }");
-            //});
+            if (userNameTextBox.Text != "")
```

Comment:
Is there a better way to handle this? maybe (!string.IsNullOrEmpty(userNameTextBox.Text))?

Reply:
When left blank, the text property for the text box has a value of " "